windows_containers/wmco_rn/windows-containers-release-notes-10-15-x.adoc

```diff
@@ -0,0 +1,29 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="windows-containers-release-notes-10-15-x"]
+= {productwinc} release notes
+include::_attributes/common-attributes.adoc[]
+:context: windows-containers-release-notes
+
+toc::[]
+
+
+The release notes for Red{nbsp}Hat OpenShift for Windows Containers tracks the development of the WMCO, which provides all Windows container workload capabilities in {product-title}.
+
+[id="about-windows-containers-numbering"]
+== WMCO numbering
```

Comment:
Should WIMCO be written out here since it is a heading? Other than that this defiantly helped!